vue/sbc-common-components/src/services/token.services.ts

```diff
@@ -92,26 +103,36 @@ class TokenServices {
         })
         .error(() => {
           clearTimeout(this.timerId)
-          console.log('refresh failed')
         })
     }, refreshInMilliSeconds)
-    console.log('[TokenServices] Timer id:' + this.timerId)
   }
 
   private syncSessionStorage () {
-    sessionStorage.setItem('KEYCLOAK_TOKEN', this.kc.token)
-    sessionStorage.setItem('KEYCLOAK_REFRESH_TOKEN', this.kc.refreshToken)
-    sessionStorage.setItem('KEYCLOAK_ID_TOKEN', this.kc.idToken)
+    if (this.kc) {
+      if (this.kc.token != null) {
```

Comment:
Do you need to explicitly check for equality to null? Maybe better just to do `if (this.kc.token) {`

Reply:
hm..the kc.token is optional param ..so have to check this explicitly... and since kc is not initialised in constructor , it will warn about object possibly null

Reply:
Lint will complain about the existing expression because you are using `!=` instead of `!==`.

You may intend the first one and the second one may break your logic. The difference is the first will compare "nullish" (equality) whereas the second will compare type (strict equality).

https://stackoverflow.com/questions/16204840/comparing-to-null-vs-in-javascript
http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html